src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -301,6 +301,36 @@ public void CheckSuccess()
             Assert.Equal("x", result.TopLevelIdentifiers.First());
         }
 
+        [Fact]
+        public void CheckSuccess2()

```

Comment:
nit: test name could be more specific, something like "CheckEqTypeMismatchWarns"

Reply:
Done